opendevin/core/config.py

```diff
@@ -123,6 +123,56 @@ def defaults_to_dict(self) -> dict:
         return dict
 
 
+@dataclass
+class SandboxConfig(metaclass=Singleton):
+    """
+    Configuration for the sandbox.
+
+    Attributes:
+        type: The type of sandbox to use. Options are: ssh, exec, e2b, local.
+        container_image: The container image to use for the sandbox.
+        user_id: The user ID for the sandbox.
+        timeout: The timeout for the sandbox.
+        initialize_plugins: Whether to initialize the plugins.
+        fast_boot: Whether to fast boot the sandbox.
+
+
+    """
+
+    type: str = 'ssh'
```

Comment:
This raises a good question... Python being python, this can work, but `type` is a keyword. It would be good to know how others think about using it as attribute name?

Reply:
type() is a function, assigning a value to this variable would overwrite its scope? 🤔 
should be avoided in any language.